etn-ccis/blui-react-component-library
# Update dev doc master branch

#### Changes proposed in this Pull Request:

- Added language direction switcher
- Added a section to refer to the React workflow package
- Dependabot stuff

#### To Test:

- Verify that the language direction switcher is working
- Verify that the workflow page exists and is pointing to the correct page.
